<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolView</class>
 <widget class="QWidget" name="toolView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>481</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="toolLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="rectBtn">
      <property name="text">
       <string>Rect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="equalEchelonBtn">
      <property name="text">
       <string>EqualEchelon</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="OrtEchelonBtn">
      <property name="text">
       <string>OrthometricEchelon</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="rhombusBtn">
      <property name="text">
       <string>Rhombus</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="squareBtn">
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="echelonBtn">
      <property name="text">
       <string>Echelon</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="parallelogramBtn">
      <property name="text">
       <string>Parallelogram</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
